<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageQueris</class>
 <widget class="QDialog" name="ManageQueris">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
       <item>
        <widget class="QListWidget" name="listWidget_StoredQueries"/>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit_ShowQuery"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btn_LoadQuery">
         <property name="text">
          <string>LOAD</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_QueryName">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_SaveQuery">
         <property name="text">
          <string>SAVE AS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_RemoveQuery">
         <property name="text">
          <string>REMOVE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_Export">
         <property name="text">
          <string>EXPORT</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_Close">
         <property name="text">
          <string>CLOSE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_Debug">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>DBG</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
